scripts/run_gaia.py

```diff
@@ -0,0 +1,25 @@
+import logging
```

Comment:
To have agents self-contained, let's move these scripts to the agent's directory.

Reply:
but this script is not specific to a single agent, but for the pair "agent + benchmark". Do you want to put it into the tapeagent dir?